var/spack/repos/builtin/packages/spectrum-mpi/package.py

```diff
@@ -0,0 +1,57 @@
+from spack import *
+
+class Spectrum(Package):
+    """IBM MPI implementation from Spectrum MPI."""
+
+    homepage = "http://www.example.com"
+    url      = "http://www.example.com/ibm-mpi-10.1.0.tar.gz"
```

Comment:
I would just remove the URL instead of having a fake one. Otherwise, when a user tries to install the package, they will see a 404 error.